playwright.config.js

```diff
@@ -0,0 +1,27 @@
+import { defineConfig, devices } from '@playwright/test';
+import { config } from 'dotenv';
+
+config({ path: './.env.test' });
+
+const PORT = process.env.PORT || 3000;
+const baseURL = `http://localhost:${PORT}`;
+
+export default defineConfig({
+  testDir: 'tests/integration/e2e',
```

Comment:
Melhor separar as pastas para os testes de integração e end-to-end

Reply:
obrigado pela sugestão, feito.